api/src/pcapi/core/object_storage/backends/gcp.py

```diff
@@ -36,6 +37,8 @@ def delete_public_object(self, folder: str, object_id: str) -> None:
             bucket = self.get_gcp_storage_client_bucket()
             gcp_cloud_blob = bucket.blob(storage_path)
             gcp_cloud_blob.delete()
+        except NotFound as not_found_exc:
+            logger.info("File not found on GCP bucket: %s", not_found_exc)
```

Comment:
(détail) Je ne retrouve pas l'erreur Sentry mentionnée dans le ticket pour voir si la représentation textuelle de l'exception contient bien le chemin de l'objet qu'on souhaite supprimer. Si c'est le cas, parfait. Sinon, on pourrait le logger comme suit : 

```suggestion
            logger.info("File not found on GCP bucket: %s", storage_path)
```

(D'ailleurs, cette version est peut-être plus concise que la représentation complète de l'exception.)